backend/scripts/monitor_ai_processing.py

```
@@ -0,0 +1,338 @@
+        
+        # 识别问题
+        issues = []
+        if self.stats["failed_summary"] > self.stats["successful_summary"]:
+            issues.append("Summary处理失败率过高")
+        if self.stats["failed_key_points"] > self.stats["successful_key_points"]:
+            issues.append("Key Points处理失败率过高")
+        if self.stats["failed_metadata"] > self.stats["successful_metadata"]:
+            issues.append("Metadata处理失败率过高")
+            
+        if issues:
+            print(f"\n⚠️  发现的问题:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)